File: Java/CreationalDesignPattern/src/main/java/com/ute/tutorialspoint/singletonpattern/SingletonPattern.java
package com.ute.tutorialspoint.singletonpattern;

public class SingletonPattern {
    private static final SingletonPattern singletonPattern = new SingletonPattern();

    private SingletonPattern() {
    }

    public static SingletonPattern getSinglePattern() {
        return singletonPattern;
    }

    public void showMessage() {
        System.out.println("Hello World!");
    }
}

File: C#/CreationalDesignPattern/CreationalDesignPattern/TutorialsPoint/SingletonPattern/SingletonPattern.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CreationalDesignPattern.TutorialsPoint
{
    public class SingletonPattern
    {
        private static SingletonPattern singletonPattern = new SingletonPattern();

        private SingletonPattern() { }

        public static SingletonPattern GetSinglePattern()
        {
            return singletonPattern;
        }

        public void showMessage()
        {
            Console.WriteLine("Hello Wolrd!");
        }
    }
}
